packages/@orbit/jsonapi/src/jsonapi-document.ts

```diff
@@ -6,12 +6,12 @@ export interface ResourceIdentity {
 }
 
 export interface ResourceHasOneRelationship {
-  data?: ResourceIdentity;
+  data?: ResourceIdentity | null;
   meta?: any;
 }
 
 export interface ResourceHasManyRelationship {
-  data?: ResourceIdentity[];
+  data?: ResourceIdentity[] | null;
```

Comment:
Not sure about this one, wouldn't an empty hasMany relationship be represented by an empty array, rather than `null`?

Reply:
Fixed. Thx!